CMakeLists.txt

```diff
@@ -121,7 +121,7 @@ if(${TRITON_ENABLE_GPU})
 
   enable_language(CUDA)
   set_target_properties(kernel_library_new PROPERTIES LANGUAGE CUDA)
-  set_target_properties(kernel_library_new PROPERTIES CUDA_ARCHITECTURES "53;60-real;62-real;70-real;72-real;75-real;80-real;86-real;89-real;90-real")
+  set_target_properties(kernel_library_new PROPERTIES CUDA_ARCHITECTURES "53;60-real;62-real;70-real;72-real;75-real;80-real;86-real;89-real;90-real;100-real;120-real")
```

Comment:
I see we define architecture codes for non supported platforms, shouldn't it be removed? 

I can't find appropriate list for iGPU platform